client/src/components/Dialogs/CartModal.js

```diff
@@ -0,0 +1,264 @@
+            </ListItemText>
+          </ListItem>
+          <ListItem>
+            <img
+              src={
+                chosenChefProfile
+                  ? chosenChefProfile.profilePictureUrl
+                  : defaultUserImage
+              }
+              alt="profileImage"
+              className={classes.img}
+            />
+          </ListItem>
+          <ListItem divider>
+            <ListItemText>
+              <Typography variant="h6">Bio</Typography>
+              <span>{chosenChefProfile && chosenChefProfile.bio}</span>
+            </ListItemText>
+            <ListItemText>
+              <Typography variant="h6">Experts In:</Typography>
+              <span>
+                {chosenChefProfile && chosenChefProfile.cuisineTags.join(",")}
+              </span>
+            </ListItemText>
+          </ListItem>
+          <ListItem dense>
+            <TableContainer>
+              <Table
+                className={classes.table}
+                size="small"
+                aria-label="a dense table"
+              >
+                <TableHead>
+                  <TableRow>
+                    <TableCell></TableCell>
+                    <TableCell align="center">Menu</TableCell>
+
+                    <Hidden smDown>
+                      <TableCell align="left">Qty</TableCell>
+                    </Hidden>
+                    <Hidden smDown>
+                      <TableCell align="center">Price</TableCell>
+                    </Hidden>
+
+                    <TableCell align="right"></TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {cart.map(({ qty, recipe }) => (
+                    <TableRow key={recipe._id}>
+                      <TableCell component="th" scope="item">
+                        <Avatar
+                          alt="Remy Sharp"
+                          src={recipe.recipePictureUrl || plate}
+                        />
+                      </TableCell>
+                      <TableCell align="justify">{recipe.name}</TableCell>
+                      <Hidden smDown>
+                        <TableCell align="justify" nowrap="nowrap">
+                          <span>{qty}</span>{" "}
+                          <ButtonGroup
+                            disableElevation
+                            variant="contained"
+                            className={classes.btnGroup}
+                          >
+                            <Button
+                              onClick={() =>
+                                qty > 1
+                                  ? decreaseCount(dispatch, {
+                                      id: recipe._id,
+                                      qty: qty - 1,
+                                      chef: recipe.user,
+                                    })
+                                  : removeFromCart(dispatch, recipe._id)
+                              }
```

Comment:
Would be good to refactor these out to handler functions and pass those to onClick, would improve readability